lib/ansible/modules/yum_repository.py

```diff
@@ -603,7 +644,7 @@ def main():
         username=dict(),
     )
 
-    argument_spec['async'] = dict(type='bool')
+    argument_spec['async'] = dict(type='bool', default=True)
```

Comment:
Is this a change in behavior to match the docs?

Reply:
Looks like that is the case.